packaging/fedora/snapd.spec

```diff
@@ -840,6 +840,13 @@ fi
 if [ "$1" -eq 2 -a "$2" -eq 1 ]; then
    # Upgrade from pre 2.38 version
    %{_libexecdir}/snapd/snap-mgmt-selinux --patch-selinux-mount-context=system_u:object_r:snappy_snap_t:s0 || :
+
+   # snapd might have created fontconfig cache directory earlier, but with
+   # incorrect context due to bugs in the policy, make sure it gets the right one
+   # on upgrade when the new policy was introduced
+   if [ -d "%{_localstatedir}/cache/fontconfig" ]; then
+      restorecon -R %{_localstatedir}/cache/fontconfig || :
```

Comment:
Curious about the `|| :` pattern used in the scriptlet, is that typical?

Reply:
Heh, I noticed the same and assumed its a RPM pattern but its good to double check.

Reply:
Yes, it's a pattern that repeats in the macros and snippets: https://fedoraproject.org/wiki/PackagingDrafts/SELinux#File_contexts